openedx_lti_tool_plugin/http.py

```diff
@@ -0,0 +1,20 @@
+"""HTTP objects for openedx_lti_tool_plugin."""
```

Comment:
I would call this module "messages" or something else, as "http" would be confusing. What do you think? @kuipumu

Reply:
@Squirrel18 I'm not sure what else called it since it's a subclass of a HTTP response object, I don't think "messages" is an appropriate name for a module that stores HTTP objects.